docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -822,6 +828,40 @@
                 </tr>
 
                 <tr>
+                    <th colspan="3" class="section" id="streams"><a href="../container/streams.html">Streams</a></th>
+                </tr>
+
+                <tr>
+                    <td class="property" id="streams-system">streams.<span class="stream">stream-id</span>.samza.system</td>
+                    <td class="default"></td>
+                    <td class="description">
+                        The <span class="system">system-name</span> of the system on which this stream will be accessed.
+                        This property binds the stream to one of the systems defined with the property
+                        systems.<span class="system">system-name</span>.samza.factory. <br>
+                        If this property isn't specified, it is inherited from job.default.system.
+                    </td>
+                </tr>
+
+                <tr>
+                    <td class="property" id="streams-physical-name">streams.<span class="stream">stream-id</span>.samza.physical.name</td>
```

Comment:
Thanks for explaining @jmakes ! I was going to ask the same question. 
I understand that it is a significant overhaul of configs to change to the new format. But having the intermediate period where we support both systems.<streamname> amd "streams.<streamname>" is going to be confusing for out users.  I am still not clear how we can cleanly transition over from the legacy configs to the new configs. Can we please document the proposed change and how the transition is going to happen? 

Additionally, the config template approach seems to increase the burden on the user to understand the config redirection. I think it arises due to this flattened config format of simply string based key-value pair. Wonder if there is a better way to do it. 

Quick question orthogonal to what we are discussing here: If the separation of logical and physical identifier for stream is aiding the HDFS use-case, how are they doing it now? Right now, does HDFS not support stream level properties?